src/App.svelte

```diff
@@ -1,7 +1,9 @@
 <script>
   import { exampleData } from "./ExampleData.js";
   export let servers = exampleData;
-  console.log(exampleData);
+  export async function grabIp(text) {
```

Comment:
this grabIP function is unnecessary since all it does is call writeText. Just pass in navigator.clipboard.writeText to on:click.